src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -226,9 +226,9 @@ public static ColumnFamily cas(String keyspaceName,
                 Tracing.trace("Reading existing values for CAS precondition");
                 long timestamp = System.currentTimeMillis();
                 ReadCommand readCommand = ReadCommand.create(keyspaceName, key, cfName, timestamp, request.readFilter());
-                ConsistencyLevel consistencyForPrecondition = consistencyForPaxos == ConsistencyLevel.LOCAL_SERIAL
+                ConsistencyLevel consistencyForPrecondition = maybeCoerceReadConsistencyLevel(consistencyForPaxos == ConsistencyLevel.LOCAL_SERIAL
```

Comment:
Do we still need the `maybeCoerceReadConsistencyLevel` here then? Considering it is used in the read later on